rules/integrations/aws/persistence_s3_obect_or_bucket_exposed.toml

```diff
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2021/06/05"
+maturity = "production"
+updated_date = "2021/06/05"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+Identifies when a S3 object or bucket ACL has been modified. This ACL exposes the object or bucket to anyone.
+"""
+false_positives = [
+    """
+    S3 ACL's may be modified by a network administrator. 
```

Comment:
I would elaborate that s3 buckets can be made public intentionally by administrators